playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,145 @@
+
+## Requirements:
+
+- [GCP project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+- [GCP User account](https://cloud.google.com/appengine/docs/standard/access-control?tab=python) _(Note: You will find the instruction "How to create User account" for your new project)_<br>
+  Ensure that the account has at least the following privileges:
+    - Service Account Admin
+    - Storage Admin
+    - Service Usage Admin
+    - Cloud Build Editor
+    - Security Admin
+    - Service Account User
+- [gcloud CLI](https://cloud.google.com/sdk/docs/install-sdk)
+- An existing GCP Bucket to save Terraform state - `state_bucket`
+- DNS name for your Playground deployment instance
+- [Terraform](https://www.terraform.io/)
+- [Apache Beam GitHub](https://github.com/apache/beam) repository cloned locally
+
+## 1. Set up the Google Cloud Build for your GCP project
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup` provisions dependencies required to set up Cloud Build for Playground:
+- Required API services
+- Cloud Build service account - `playground-cloudbuild-sa`
+- IAM permissions for Cloud Build service account - `playground-cloudbuild-sa`
+
+#### To execute the module:
+
+1. Set following environment variables:
+   - `STATE_BUCKET`: GCP Storage bucket name to save Terraform state
+   - `GOOGLE_PROJECT`: GCP Project ID
+   - `GOOGLE_REGION`: GCP region to save resources to
+   - `ENVIRONMENT_NAME`: Environment where Playground will be deployed (located at playground/terraform/environment/environment_name)
+   - `DNS_NAME`: DNS for deployed Playground webpage
+   - `NETWORK_NAME`: GCP VPC Network Name for Playground deployment
+   - `GKE_NAME`: Playground GKE Cluster name
+   - `TAG`: Tag for Playground images
+   - `DOCKER_REPOSITORY_ROOT`: GCP Artifact Registry repository name to store Playground container images
+
+```console
+    export STATE_BUCKET="state_bucket" \
+    GOOGLE_PROJECT="project_id" \
+    GOOGLE_REGION="region" \
+    ENVIRONMENT_NAME="env_name" \
+    DNS_NAME="dns_name" \
+    NETWORK_NAME="network_name" \
+    GKE_NAME="gke_name" \
+    TAG="tag" \
+    DOCKER_REPOSITORY_ROOT="us-central1-docker.pkg.dev/playground-repository"
+```
+**Note:**  Please see [Cloud Build locations](https://cloud.google.com/build/docs/locations)     for the list of all supported locations.
+
+2. Run commands:
+
+
+```console
+# Create a new authentication configuration for GCP Project with the created user account
+gcloud init
+
+# Command imports new user account credentials into Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to 01.setup folder
+cd playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/
+
+# Run terraform scripts
+terraform init -backend-config="bucket=$STATE_BUCKET"
+terraform apply -var "project_id=$GOOGLE_PROJECT"
```

Comment:
-var "region=$GOOGLE_REGION"

Reply:
It should not, latest changes have to be pulled @rshamunov